bin/workflows/pycbc_create_offline_search_workflow

```diff
@@ -551,12 +563,101 @@ for inj_file, tag in zip(inj_files, inj_tags):
                                                      output_dir,
                                                      tags=combctags)
     
+    # FIXME: Should this be censored_veto
     found_inj = wf.find_injections_in_hdf_coinc\
         (workflow, [combined_inj_bg_file], [file_for_injfind],
          final_veto_file[0], final_veto_name[0], output_dir, tags=combctags)
     
     inj_coincs += [combined_inj_bg_file]
 
+    # Set files for plots
+    found_inj_dict[tag] = found_inj
+    insps_dict[tag] = insps
+    small_inj_file_dict[tag] = small_inj_file
+
+# Make combined injection file
+# FIXME: Should this be censored_veto?
+if len(files_for_combined_injfind) > 0:
+    found_inj_comb = wf.find_injections_in_hdf_coinc\
+        (workflow, inj_coincs, files_for_combined_injfind, final_veto_file[0],
+         final_veto_name[0], 'allinj', tags=['ALLINJ'])
+
+
+############################ Injection plots #################################
+
+# Per injection run plots
+for inj_file, tag in zip(inj_files, inj_tags):
+    s = []
+    found_inj = found_inj_dict[tag]
+    insps = insps_dict[tag]
+    small_inj_file = small_inj_file_dict[tag]
+    injdir = rdir['injections/%s' % tag]
+    sensdir = rdir['search_sensitivity/%s' % tag]
+
+    # FIXME: Ian has some issues with how these sensitivity jobs are created
```

Comment:
@spxiwh can you convert this FIXME into a github issue and explain more there?  It's a bit too cryptic here.

Reply:
I've just removed this FIXME for now. Maybe at some point we can try to unify how the various plotting codes are called, but I don't think that will happen any time soon.